sections/semantics-forms.include

```diff
@@ -1584,6 +1584,7 @@
     <dd><{input/autocapitalize}> - Hint for helping to capitalize user input</dd>
     <dd><{input/autocomplete}> - Hint for form autofill feature</dd>
     <dd><{input/autofocus}> - Automatically focus the form control when the page is loaded</dd>
+    <dd><{input/capture}> - Specifies the preferred facing mode for the media capture mechanism, defined in [[!html-media-capture]]</dd>
```

Comment:
There should be a direct link for the attribute. As mentioned offline, I will make a PR against your branch to do that.